### PR TITLE
L2 774 skeleton project card

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Create the skeleton that reflects the content of the project and proposal card

[L2-774](https://dfinity.atlassian.net/browse/L2-774)

https://user-images.githubusercontent.com/98811342/178540751-d29cc7c2-3897-40ae-b30a-05a93ab1bc66.mp4

# Changes

- SkeletonProjectCard
- SkeletonProposalCard
- SkeletonLogo

# Tests

No logic changes

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
LGTM!

File: frontend/svelte/src/lib/components/ui/SkeletonProjectCard.svelte [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!-- adapted source: https://github.com/ionic-team/ionic-framework/tree/main/core/src/components/skeleton-text -->
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/SkeletonProjectCard.svelte:1]: you can remove this comment here

File: frontend/svelte/src/lib/components/ui/SkeletonProposalCard.svelte [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!-- adapted source: https://github.com/ionic-team/ionic-framework/tree/main/core/src/components/skeleton-text -->
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/SkeletonProposalCard.svelte:1]: likewise

File: frontend/svelte/src/lib/components/ui/SkeletonProjectCard.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/SkeletonProjectCard.svelte]: Thanks, done.

**Pull request merged**